### PR TITLE
Adding node status to top node and adding more statuses (v1.NodeConditionType)

### PR DESCRIPTION
**Pull request by moshe-kabala**

### CODE REVIEW

File: cmd/util/node.go [hunk 1]
```diff
@@ -44,11 +44,17 @@ func GetNodeInternalAddress(node v1.Node) string {
 	return "unknown"
 }
 
-func IsNodeReady(node v1.Node) bool {
+func GetNodeStatus(node v1.Node) v1.NodeConditionType {
 	for _, condition := range node.Status.Conditions {
-		if condition.Type == v1.NodeReady && condition.Status == v1.ConditionTrue {
-			return true
+		if condition.Status == v1.ConditionTrue {
+			return condition.Type
```
Comment (omer-dayan) [inline cmd/util/node.go:50]: Why would we limit ourself to a single status?
There can be more that one. For example MemoryPressure and NetworkUnavailable are unrelated and can happen at the same time
Lets show all of them

File: cmd/util/node.go [hunk 2]
```diff
@@ -44,11 +44,17 @@ func GetNodeInternalAddress(node v1.Node) string {
...
 		}
 	}
```
Comment (omer-dayan) [inline cmd/util/node.go:52]: Screen Shot 2021-04-20 at 19 07 26

Notice this, we may need to check it either

Comment (omer-dayan) [reply cmd/util/node.go:52]: (Not sure about that)

File: cmd/util/node.go [hunk 3]
```diff
@@ -44,11 +44,17 @@ func GetNodeInternalAddress(node v1.Node) string {
...
-	return false
+	return v1.NodeConditionType(v1.ConditionUnknown)
+}
+
+func IsNodeReady(node v1.Node) bool {
```
Comment (omer-dayan) [inline cmd/util/node.go:56]: Notice please that there is a file named 'tensoboard.go' that use this function. 
However no one use the file anywhere, lets delete it.

Comment (omer-dayan) [inline cmd/util/node.go:56]: Screen Shot 2021-04-20 at 19 12 31

Furthermore, this is the only place where we really use this function.
Notice that it rely on the assumption that if node is not in ready state then it has no pods on it. Which is not true in your case.

**Review by omer-dayan — COMMENTED** [review]
Really good job..

Generally - when you touch the CLI,
תסתכל קצת מסביב לקוד שאתה כותב, יש מלא פונקציות שלא משתמשים בהם, יש המון דברים שכבר לא בשימוש, מלא הערות לא רללונטיות, אל תפחד למחוק

File: cmd/util/node.go (hunk 3 above)
Comment (moshe-kabala) [reply cmd/util/node.go:56]: It looks that if the node is not in ready state then it has to **active** pod. Anyway, this is beyond the bug scope.

Comment (moshe-kabala) [reply cmd/util/node.go:56]: so delete the entire file(tensoboard.go)?

File: cmd/util/node.go (hunk 1 above)
Comment (moshe-kabala) [reply cmd/util/node.go:50]: Good to know, I will join them (State1, State2)

File: cmd/util/node.go (hunk 2 above)
Comment (moshe-kabala) [reply cmd/util/node.go:54]: I added Unschedulable into the status

**Review by omer-dayan — APPROVED** [review]

**Pull request merged**